app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum([player.get_rating() for player in team])
```

Comment:
You do not need to convert to a list
```suggestion
    return sum(player.get_rating() for player in team)
```